### DISCUSSION TITLE
Re:   Approving Pull Request

### DISCUSSION THREAD

**Discussion by sanvidpunde** [sanvidpunde/MyUniLifeSimulation#160] [General]
Hello Teammates,

Please be sure to approve the PR after reviewing the code and all the other files.
As this can become a blocker.
Thanks for understanding.

**Comment by RahulKrishnaGajula** [comment]
Yes, this is a very important aspect of maintaining a git repository. 

All members have to be active and take initiative to check the repository (Alerts are sent whenever there is a new PR to everyone's emails).

It is not the responsibility of the PR-creator to constantly ask others to approve the PR

**Reply by sanvidpunde** [reply]
Agreed,
this is becoming a huge blocker for final deployment.
